vars/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+redis_unixsocket: /var/run/redis/redis.sock
+redis_unixsocketperm: 770
+redis_port: 0
```

Comment:
Port 0 isn't valid

Reply:
yes port 0 is valid for redis if you set port to 0 redis will only connect on the unix socket